aws/v12.5.0/release.diff

```diff
@@ -0,0 +1,69 @@
+    version: 1.1.1                                                     version: 1.1.1
+  - componentVersion: 0.3.6                                          - componentVersion: 0.3.6
+    name: metrics-server                                               name: metrics-server
+    version: 1.1.1                                                     version: 1.1.1
+  - name: net-exporter                                               - name: net-exporter
+    version: 1.9.0                                                     version: 1.9.0
+  - componentVersion: 1.0.1                                          - componentVersion: 1.0.1
+    name: node-exporter                                                name: node-exporter
+    version: 1.3.0                                                     version: 1.3.0
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 2.3.2                                                     version: 2.3.2
+  - name: aws-cni                                                    - name: aws-cni
+    version: 1.7.3                                                     version: 1.7.3
+  - name: aws-operator                                               - name: aws-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 9.1.1                                              |      version: 9.1.2
+  - name: calico                                                     - name: calico
+    version: 3.15.3                                                    version: 3.15.3
+  - name: cert-operator                                              - name: cert-operator
+    reference: 0.1.0-2                                                 reference: 0.1.0-2
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 0.1.0                                                     version: 0.1.0
+  - name: cluster-operator                                           - name: cluster-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 3.2.0                                              |      version: 3.3.1
+  - name: containerlinux                                             - name: containerlinux
+    version: 2512.5.0                                                  version: 2512.5.0
+  - name: etcd                                                       - name: etcd
+    version: 3.4.13                                                    version: 3.4.13
+  - name: kubernetes                                                 - name: kubernetes
+    version: 1.17.12                                                   version: 1.17.12
```

Comment:
diff is missing 1.17.13

Reply:
Good catch! I regenerated the release.diff file.